Phoenix/hello/lib/hello_web/controllers/fallback_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HelloWeb.FallbackController do
```

Comment:
@samu975 Please, explain how this fallback controller works.

Reply:
is used to handle failed events. In this case, it was used to handle an error not found and an unprocessable_entity error.